scipy/stats/tests/test_qmc.py

```diff
@@ -212,15 +212,16 @@ def test_update_discrepancy(self):
             update_discrepancy(x_new, space_1[:-1], disc_init)
 
     def test_perm_discrepancy(self):
-        seed = np.random.RandomState(123456)
-        qmc_gen = qmc.LatinHypercube(5, seed=seed)
+        qmc_gen = qmc.LatinHypercube(5)
         sample = qmc_gen.random(10)
         disc = qmc.discrepancy(sample)
 
+        rng = np.random.default_rng(46449423132557934943847369749645759997)
+
         for i in range(100):
```

Comment:
@mdhaber I went through the file again and here is the only other case where it could have an impact. The rest is checking shapes (`MultivariateNormalQMC` tests), raises/warnings, or from deterministic sequences (and here seeding would is not used/called).
```suggestion
        rng = np.random.default_rng(46449423132557934943847369749645759997)
        qmc_gen = qmc.LatinHypercube(5, seed=rng)
        sample = qmc_gen.random(10)
        disc = qmc.discrepancy(sample)

        for i in range(100):
```